extensions/binding/org.eclipse.smarthome.binding.sonos/README.md

```diff
@@ -79,6 +84,14 @@ The devices support the following channels:
 | zonegroupid | String | Id of the Zone Group the Zone Player belongs to | all |
 | zonename | String | Name of the Zone Group the Zone Player belongs to | all |
 
+## Audio Support
+
+All supported Sonos devices are registered as an audio sink in the framework.
+Audio streams are treated as notifications, i.e. they are fed into the `notificationsound` channel and changing the volume of the audio sink will change the `notificationvolume`, not the `volume`.
+Note that the Sonos binding has a limit of 20 seconds for notification sounds. Any sound that is longer than that will be cut off.
+
+An exception is done if an URL audio stream is passed (e.g. an Internet radio stream). This will be handled by the `playuri` channel.
```

Comment:
Is the situation simply that file system audio streams get sent as a Sonos notification whereas url audio streams get sent down the playuri channel?